app/src/modules/settings/routes/roles/item/item.vue

```diff
@@ -152,7 +152,8 @@ export default defineComponent({
 
 		const { edits, hasEdits, item, saving, loading, error, save, remove, deleting, isBatch } = useItem(
 			ref('directus_roles'),
-			primaryKey
+			primaryKey,
+			{ deep: { users: { _limit: 0 } } }
```

Comment:
Creative use of deep to keep the field in the result. 👍 
There's a type issue with `_limit` in this instance.
Could we add a new `QueryDeep` type to cover the `_` prefixed fields in `deep`? 🤔 

<img width="243" alt="image" src="https://user-images.githubusercontent.com/26413686/200108943-82652eac-c900-4dcc-82d5-75dbeae41f34.png">